package.json

```diff
@@ -51,8 +51,7 @@
   },
   "lint-staged": {
     "**/*.{js,ts}": [
-      "eslint --fix",
-      "git add"
```

Comment:
why we remove `"git add"` ?

Reply:
This is the main [breaking change](https://github.com/okonet/lint-staged#v10) `lint-staged` added in v10